pkg/metadata/labels.go

```diff
@@ -135,12 +134,10 @@ func NewInstanceLabeler(instanceMeta metav1.Object) GenericLabeler {
 // KroVersion, and ControllerPodID labels on a resource.
 func NewKroMetaLabeler(
 	kroVersion string,
-	controllerPodID string,
 ) GenericLabeler {
 	return map[string]string{
-		OwnedLabel:           "true",
-		KroVersionLabel:      kroVersion,
-		ControllerPodIDLabel: controllerPodID,
```

Comment:
Just to confirm. We are removing "controller-pod-id: kro-pod" from the label ?

Reply:
Yes, it was hard coded, and I don't think it's bringing any value :)